src/modules/services_manager_service.rb

```diff
@@ -293,11 +294,38 @@ def start_modes(name)
 
     # Returns full information about the service as returned from systemctl command
     #
+    # Including its socket's information when the service supports starting on demand
+    #
+    # @param service [Yast2::SystemService]
+    #
+    # @return [String] full unformatted information
+    def status(service)
+      output = service_status(service.name)
+
+      return output unless service.support_start_on_demand?
+
+      socket = Yast2::Systemd::Socket.for_service(service.name)
```

Comment:
I have to say that this solution does not finish to convince to myself.

`Yast2::SystemService` already knows the associated socket, but it is a [private method](https://github.com/yast/yast-yast2/blob/b6638bdffe2309e1b40a62ec2a1c6892e277a097/library/systemd/src/lib/yast2/system_service.rb#L548). On the other hand, the `Yast2::Systemd::Service` object (reachable through _service.service_) has the [socket accessible publicly](https://github.com/yast/yast-yast2/blob/b6638bdffe2309e1b40a62ec2a1c6892e277a097/library/systemd/src/lib/yast2/systemd/service.rb#L195), but using it from here (`service.service.socket) could over complicate the unit test.

I also discarded to give an `Yast2::Systemd::Service` as argument because I think that it is better to rely on `#support_start_on_demand?` instead of using  `service.socket` directly.

Do you think that worth it to make  `Yast2::SystemService#socket` public in order to keep things simple? Make sense for you? 

Any other ideas?